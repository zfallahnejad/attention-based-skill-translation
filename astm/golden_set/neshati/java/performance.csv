22656,performance
829571,performance
1103872,performance
571407,performance
605744,performance
335858,performance
70604,performance
869736,performance
179850,performance
276052,performance
3182664,performance
21234,performance
3448419,performance
