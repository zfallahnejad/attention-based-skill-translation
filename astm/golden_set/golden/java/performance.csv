22656,51,59,0.4636363636363636
829571,28,34,0.45161290322580644
1103872,26,38,0.40625
571407,25,26,0.49019607843137253
605744,19,12,0.6129032258064516
335858,18,13,0.5806451612903226
70604,17,14,0.5483870967741935
179850,15,10,0.6
869736,15,19,0.4411764705882353
276052,13,6,0.6842105263157895
21234,12,11,0.5217391304347826
3448419,12,9,0.5714285714285714
3182664,12,5,0.7058823529411765
